Codeinwp/otter-blocks
# Pro patterns

Closes #.


### Summary


This adds Pro patterns to Otter. We can test them first and then also add few of these to Otter free with some sort of promotional mechanism.

Can you help with the testing @JohnPixle @mghenciu?

### Screenshots 

----

### Test instructions


If you install just Otter free, you should see Upsell notices on all the Otter Patterns once you insert them. If one disable it once then it won't show again in the editor session.

If you dismiss them in 3 sessions, they won't appear again for a month. You can clear browser storage to make them appear again. Run this in the console:

`window.localStorage.clear();`

And if you install Otter Pro & activate the license, you should start seeing new Patterns from Otter Pro: https://demosites.io/otter/pro-patterns/



---- 

### Checklist before the final review
